File: TaskTracker.Core/src/Models/PostRequests/IssueEstimatePredictionPR.cs
using TaskTracker.Core.src.Enums;

namespace TaskTracker.Core.src.Models.PostRequests
{
    public class IssueEstimatePredictionPR : BasePostRequest
    {
        public int? Id { get; set; }

        public string Name { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public IssueType Type { get; set; }

        public IssueStatus Status { get; set; }

        public IssuePriority Priority { get; set; }

        public int? ParentId { get; set; }

        public int? AssigneeId { get; set; }

        public int ProjectId { get; set; }
    }
}

File: ui/task-tracker/src/app/shared/model/postRequests/issueEstimatePredictionPR.ts
import { IssuePriority } from "../../enums/issue-priority";
import { IssueStatus } from "../../enums/issue-status";
import { IssueType } from "../../enums/issue-type";

export class IssueEstimatePredictionPR {
    id: number | undefined;
    name!: string;
    description!: string;
    type!: IssueType;
    status!: IssueStatus;
    priority!: IssuePriority;
    parentId: number | undefined;
    assigneeId: number | undefined;
    projectId!: number;
}
